ui/StatusQ/src/StatusQ/Controls/StatusPinInput.qml

```diff
@@ -226,9 +229,15 @@ Item {
 
             // Some component validations:
             if(text.length !== d.currentPinIndex)
-                console.error("StatusPinInput input management error. Current pin length must be "+ text.length + "and is " + d.currentPinIndex)
+                console.error("StatusPinInput input management error. Current pin length must be "+ text.length + " and is " + d.currentPinIndex)
```

Comment:
Does QML support strings using backticks?
If so, this would be cleaner:
```suggestion
                console.error(`StatusPinInput input management error. Current pin length must be ${text.length} and is ${ d.currentPinIndex}`)
```

Reply:
Yeah it does; existing code so didn't touch it more than I wanted :)